pkg/monitoring/virt-controller/metrics/component_metrics.go

```diff
@@ -0,0 +1,51 @@
+package metrics
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	ioprometheusclient "github.com/prometheus/client_model/go"
+)
+
+var (
+	operatorMetrics = []operatormetrics.Metric{
+		virtControllerLeading,
+		virtControllerReady,
+	}
+
+	virtControllerLeading = operatormetrics.NewGauge(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_virt_controller_leading",
+			Help: "Indication for an operating virt-controller.",
+			ExtraFields: map[string]string{
+				"StabilityLevel": "STABLE",
```

Comment:
@machadovilaca Hi. Do we have an upstream documentation about our metrics graduation policy?